insonmnia/hub/server.go

```diff
@@ -42,7 +43,10 @@ type Hub struct {
 	tasksmu sync.Mutex
 	tasks   map[string]string
 
-	wg sync.WaitGroup
+	wg        sync.WaitGroup
+	startTime time.Time
+	publicIP  string
```

Comment:
What's the meaning of this? To ask Hub's public IP I should know its public ip, isn't it?